torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -338,6 +339,17 @@ at::Tensor AtenXlaType::avg_pool2d(const at::Tensor& self,
       XlaHelpers::I64List(padding), count_include_pad));
 }
 
+at::Tensor AtenXlaType::adaptive_avg_pool2d(const at::Tensor& self,
+                                            at::IntArrayRef output_size) const {
+  if (self.dim() != 4 ||
+      !IsSupportedAdaptiveAvgPool2d(XlaHelpers::I64List(self.sizes()),
+                                    XlaHelpers::I64List(output_size))) {
+    return AtenXlaTypeBase::adaptive_avg_pool2d(self, output_size);
```

Comment:
Is this an operation downgrade?
If yes, we better scream about it.

Reply:
What do you mean by downgrade? If the output size doesn't divide the input size, this doesn't have a good XLA lowering -- the stride isn't constant. And `AtenXlaTypeBase::adaptive_avg_pool2d` will log, not sure how much harder we should scream.

Reply:
Oh OK.
I thought it we were downgrading/shortcutting for something we did not want to lower yet.